### PR TITLE
Added Rating Component

### PR DESCRIPTION
**Pull request by mansi2011**

### CODE REVIEW

**Comment by mansi2011** [comment]
Added Rating Component

Rating

File: Components/Rating/Rating.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                        style="width: 1024px; height: 300px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+
+            </div>
+            
+
```
Comment (neetukumari4858) [inline Components/Rating/Rating.html:96]: remove Extra space

**Review by neetukumari4858 — APPROVED** [review]
- UI looks good.
- code is neat & clean.

File: Components/Rating/Rating.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
+.favorites{
+    /* border: 1px solid var(--black-font-color); */
```
Comment (shivanigangadharan) [inline Components/Rating/Rating.css]: Can remove comments if not required

**Review by shivanigangadharan — APPROVED** [review]
UI looks good, code is neatly structured as well. Can merge this 👍🏼

File: Components/Rating/Rating.css (hunk 2 above)
Comment (mansi2011) [reply Components/Rating/Rating.css]: done

**Pull request merged**